### ISSUE TITLE
where is the collect.sh

### ISSUE THREAD

**Issue by gutongjiang** [m2049r/xmrwallet#617]

**Comment by m2049r** [comment]
i think you are looking for install.sh?

**Comment by binaryFate** [comment]
The very end of `doc/BUILDING-external-libs.md` says:

```
cd <path-to-xmrwallet>/external-libs
 # remove old stuff
find . -name "*.a" -or -name "*.h" -type f -delete
./collect.sh
```

But collect.sh does not seem to be in the git repo at all.

**Comment by m2049r** [comment]
yes, the whole build system has been replaced by the `Makefile` and the scripts in the `script` folder, both under `external-libs`.
The docs need to be adapted.

**Comment by netnye** [comment]
I'm not a programmer, but I'd be willing to help the project by helping with updating documentation. How do I get started?

**Comment by HashEngineering** [comment]
@m2049r I was trying to build this app by following your comment here:  https://github.com/m2049r/xmrwallet/issues/617#issuecomment-541361126, but I am having some trouble, perhaps because of the Android NDK I am using.  Can you do we need to us `r16b`?
